sphinx/builders/html/__init__.py

```diff
@@ -271,6 +272,26 @@ def init_highlighter(self) -> None:
             style = 'sphinx'
         self.highlighter = PygmentsBridge('html', style)
 
+        if self.config.html_pygments_dark_style is not None:
```

Comment:
I noticed the document for this option is needed.

Reply:
Tests are also needed.

Reply:
Yes I will add them

Reply:
Documentation and tests added